dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/DateInterval.java

```diff
@@ -35,13 +35,18 @@ public DateInterval(Date beginTime, Date endTime){
 
     @Override
     public boolean equals(Object obj) {
-        try{
-            DateInterval dateInterval = (DateInterval) obj;
-            return startTime.equals(dateInterval.getStartTime()) &&
-                    endTime.equals(dateInterval.getEndTime());
-        }catch (Exception e){
+        if (obj != null) {
```

Comment:
why use try/catch surround equals method?

Reply:
check whether obj is null